### PR TITLE
Update docs for release v0.3

### PR DESCRIPTION
**Pull request by flaviodsr**
Update the documentation to reflect the changes made for the next
FuseML release. Also includes fixing some typos and formating.

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -47,61 +47,54 @@ You should get an output similar to this:
...
+## 4. Ensure FuseML components are up and running
 
 ```bash
-# Let's check the fuseml-core component
-kubectl get pods -n fuseml-core -o wide
-# Now let's check gitea
-kubectl get pods -n gitea -o wide
-# Let's check tekton
-kubectl get pods -n tekton-pipelines -o wide
+eval 'kubectl get pods -n '{istio-system,gitea,fuseml-registry,tekton-pipelines,fuseml-core}' -o wide;'
```
Comment (jsuchome) [inline docs/quickstart.md:97]: Hm, while this is nice one-liner, I like the parts with more explanations more...

**Review by jsuchome — DISMISSED** [review]

File: docs/quickstart.md (hunk 1 above)
Comment (flaviodsr) [reply docs/quickstart.md:97]: ok. I can revert that change and include the missing services to be checked.

Comment (flaviodsr) [reply docs/quickstart.md]: done!

Comment (jsuchome) [reply docs/quickstart.md]: thanks!

**Review by jsuchome — DISMISSED** [review]

File: docs/tutorials/kserve-basic.md [hunk 2]
```diff
@@ -80,13 +79,13 @@ Use the example workflow definition to create a workflow in FuseML:
 fuseml workflow create workflows/mlflow-e2e.yaml
 ```
 
-**6.** Assign the codeset to the workflow
+## 6. Assign the workflow to the codeset
```
Comment (stefannica) [inline docs/tutorials/kserve-basic.md:82]: Not sure I agree with you here. And I'm not the only one, remember https://github.com/fuseml/fuseml/issues/230 :-)

File: docs/tutorials/seldon-core.md [hunk 3]
```diff
@@ -92,17 +92,17 @@ inputs:
     default: auto
 ```
 
-That `predictor` option is set to `auto` value - this means the predictor engine will be set automatically based on the properties 
+That `predictor` option is set to `auto` value - this means the predictor engine will be set automatically based on the properties
 of the model. With the model we are using in this tutorial, the prediction value will be set to 'sklearn' value and Seldon Core will use
 SKLearn server. You could change the default value to `tensorflow` or `triton` to use different inference servers; that is, if your model supports it.
 
-**6.** Assign the codeset to the workflow
+## 6. Assign the workflow to the codeset
```
Comment (stefannica) [inline docs/tutorials/seldon-core.md:99]: Same thing here

**Review by stefannica — APPROVED** [review]
A special thank you for taking the time to update those CLI command outputs. I only have one tiny comment, but you can safely ignore it if you wish to.

File: docs/tutorials/kserve-basic.md (hunk 2 above)
Comment (flaviodsr) [reply docs/tutorials/kserve-basic.md:82]: Well, I still think that workflow is assigned to codesets 😄 ... We never reached a conclusion, so the minimum we can do is be consistent, and that change make it consistent with https://github.com/flaviodsr/docs/blob/update_docs/docs/tutorials/seldon-core.md?plain=1#L107

**Review by stefannica — APPROVED** [review]

**Pull request merged**